toolbox/gui/figure_3d.m

```diff
@@ -1247,9 +1247,10 @@ function SetStandardView(hFig, viewNames)
     % Displaying a surface: Load the SCS field from the MRI
     if isempty(Ranat) && ~isempty(TessInfo) && ~isempty(TessInfo(1).SurfaceFile)
         % Get subject
-        sSubject = bst_get('SurfaceFile', TessInfo(1).SurfaceFile);
+        sSubject = db_get('SurfaceFile', TessInfo(1).SurfaceFile);
+        SAnatomy = db_get('AnatomyFile', sSubject.iAnatomy, 'FileName');

```

Comment:
Typo: sAnatomy

Reply:
Fixed in 3834c6560c3d94138f25784b7f976ee4029b297a